src/ActionsMenuItem.scss

```diff
@@ -0,0 +1,30 @@
+.ActionsMenuItem__button {
+    width: 4rem;
+    float: right;
+    border: 0;
+    background-color: inherit;
+}
+
+.ActionsMenuItem__button-label {
+    margin-left: 0.5rem;
+}
+
+.ActionsMenuItem {
+    border-top: 1px solid #cccccc;
+    clear: both;
+    margin: 0.5rem;
```

Comment:
Setting margin to all sides creates uneven gap at the top most item, so let's apply margin to the sides only and adjust vertical margins from children components.

`
.ActionsMenuItem {
    border-top: 1px solid #cccccc;
    clear: both;
    margin: 0 0.5rem;
    white-space: nowrap;
}
`

Reply:
Done.